src/scos_tekrsa/configs/actions-500-600/CBRS_IQ_15dB_Attenuation.yml

```diff
@@ -2,7 +2,7 @@ stepped_frequency_time_domain_iq:
   # 500 ms captures every 10 MHz
   name: CBRS_IQ_15dB_Attenuation
   rf_path: antenna
-  reference_level: -25
+  reference_level: -20
```

Comment:
I think this is supposed to be -10

Reply:
Good catch, fixed in [47f469a](https://github.com/NTIA/scos-tekrsa/pull/39/commits/47f469a0e8725c8d30ea821a0166f0d31b0871bd)